backends/graphics/gles-custom/gles-custom.cpp

```diff
@@ -0,0 +1,703 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+// Needed so we don't get errors when including the Raspberry Pi headers
+#define FORBIDDEN_SYMBOL_ALLOW_ALL
```

Comment:
I don't use any forbidden symbols myself, but VideoCore libs seem to do so.
This is what happens when I comment out that define:

´´´
pi@raspberrypi ~/src/scummvm_BUILD $ make VERBOSE_BUILD=1
fatal: No names found, cannot describe anything.
mkdir -p backends/graphics/gles-custom/.deps
g++ -MMD -MF "backends/graphics/gles-custom/.deps/gles-custom.d" -MQ "backends/graphics/gles-custom/gles-custom.o" -MP -Wall -I/usr/include/SDL -D_GNU_SOURCE=1 -D_REENTRANT   -I/usr/include/freetype2 -mfpu=vfp -mfloat-abi=hard -march=armv6zk -mtune=arm1176jzf-s -I/opt/vc/include -I/opt/vc/include/interface/vmcs_host/linux/ -I/opt/vc/include/interface/vcos/pthreads -Wno-long-long -Wno-multichar -Wno-unknown-pragmas -Wno-reorder -Wpointer-arith -Wcast-qual -Wshadow -Wnon-virtual-dtor -Wwrite-strings -fno-exceptions -fcheck-new -DSCUMMVM_REVISION=\"-dirty\" -DHAVE_CONFIG_H -DRELEASE_BUILD -DARM_TARGET -DSDL_BACKEND -DPOSIX -DDATA_PATH=\"/usr/local/share/scummvm\" -DPLUGIN_DIRECTORY=\"/usr/local/lib/scummvm\" -DENABLE_SCUMM=STATIC_PLUGIN -DENABLE_SCUMM_7_8 -DENABLE_HE -I. -I. -I./engines -I/usr/include/SDL -D_GNU_SOURCE=1 -D_REENTRANT   -I/usr/include/freetype2 -c backends/graphics/gles-custom/gles-custom.cpp -o backends/graphics/gles-custom/gles-custom.o
In file included from /usr/include/pthread.h:24:0,
                 from /opt/vc/include/interface/vcos/pthreads/vcos_platform.h:47,
                 from /opt/vc/include/interface/vcos/vcos.h:116,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/usr/include/time.h:189:27: error: macro "clock" passed 1 arguments, but takes just 0
 extern clock_t clock (void) __THROW;
                           ^
distcc[12945] ERROR: compile backends/graphics/gles-custom/gles-custom.cpp on 192.168.5.7 failed
distcc[12945] (dcc_build_somewhere) Warning: remote compilation of 'backends/graphics/gles-custom/gles-custom.cpp' failed, retrying locally
distcc[12945] Warning: failed to distribute backends/graphics/gles-custom/gles-custom.cpp to 192.168.5.7, running locally instead
In file included from /usr/include/pthread.h:24:0,
                 from /opt/vc/include/interface/vcos/pthreads/vcos_platform.h:47,
                 from /opt/vc/include/interface/vcos/vcos.h:116,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/usr/include/time.h:189:27: error: macro "clock" passed 1 arguments, but takes just 0
 extern clock_t clock (void) __THROW;
                           ^
In file included from /usr/include/features.h:374:0,
                 from /usr/include/stdio.h:27,
                 from ./common/scummsys.h:127,
                 from ./backends/graphics/opengl/opengl-sys.h:29,
                 from ./backends/graphics/opengl/opengl-graphics.h:26,
                 from ./backends/graphics/gles-custom/gles-custom.h:26,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/usr/include/time.h:189:29: error: expected initializer before 'throw'
 extern clock_t clock (void) __THROW;
                             ^
In file included from ./common/scummsys.h:423:0,
                 from ./backends/graphics/opengl/opengl-sys.h:29,
                 from ./backends/graphics/opengl/opengl-graphics.h:26,
                 from ./backends/graphics/gles-custom/gles-custom.h:26,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./common/forbidden.h:57:89: error: expected initializer before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h: In function 'int vcos_strcasecmp(const char*, const char*)':
./common/forbidden.h:57:38: error: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' was not declared in this scope
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                      ^
./backends/platform/sdl/sdl-sys.h:69:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strcasecmp     FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:596:11: note: in expansion of macro 'strcasecmp'
    return strcasecmp(s1,s2);
           ^
./common/forbidden.h:57:89: error: expected ';' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./backends/platform/sdl/sdl-sys.h:69:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strcasecmp     FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:596:11: note: in expansion of macro 'strcasecmp'
    return strcasecmp(s1,s2);
           ^
./common/forbidden.h:57:89: error: 'SYMBOL' was not declared in this scope
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./backends/platform/sdl/sdl-sys.h:69:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strcasecmp     FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:596:11: note: in expansion of macro 'strcasecmp'
    return strcasecmp(s1,s2);
           ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h: In function 'int vcos_strncasecmp(const char*, const char*, size_t)':
./common/forbidden.h:57:38: error: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' was not declared in this scope
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                      ^
./backends/platform/sdl/sdl-sys.h:74:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strncasecmp    FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:601:11: note: in expansion of macro 'strncasecmp'
    return strncasecmp(s1,s2,n);
           ^
./common/forbidden.h:57:89: error: expected ';' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./backends/platform/sdl/sdl-sys.h:74:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strncasecmp    FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:601:11: note: in expansion of macro 'strncasecmp'
    return strncasecmp(s1,s2,n);
           ^
./common/forbidden.h:57:89: error: 'SYMBOL' was not declared in this scope
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
./backends/platform/sdl/sdl-sys.h:74:24: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define strncasecmp    FORBIDDEN_SYMBOL_REPLACEMENT
                        ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform.h:601:11: note: in expansion of macro 'strncasecmp'
    return strncasecmp(s1,s2,n);
           ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/vcos_logging.h: At global scope:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_logging.h:129:131: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_log_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, ...) VCOS_FORMAT_ATTR_(printf, 3, 4);
                                                                                                                                   ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_logging.h:129:113: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_log_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, ...) VCOS_FORMAT_ATTR_(printf, 3, 4);
                                                                                                                 ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_logging.h:134:141: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_vlog_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, va_list args) VCOS_FORMAT_ATTR_(printf, 3, 0);
                                                                                                                                             ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_logging.h:134:123: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_vlog_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, va_list args) VCOS_FORMAT_ATTR_(printf, 3, 0);
                                                                                                                           ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_logging.h:146:149: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_vlog_default_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, va_list args) VCOS_FORMAT_ATTR_(printf, 3, 0);
                                                                                                                                                     ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_logging.h:146:131: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_vlog_default_impl(const VCOS_LOG_CAT_T *cat, VCOS_LOG_LEVEL_T _level, const char *fmt, va_list args) VCOS_FORMAT_ATTR_(printf, 3, 0);
                                                                                                                                   ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_cmd.h:79:107: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_error( VCOS_CMD_PARAM_T *param, const char *fmt, ... ) VCOS_FORMAT_ATTR_(printf, 2, 3);
                                                                                                           ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_cmd.h:79:89: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_error( VCOS_CMD_PARAM_T *param, const char *fmt, ... ) VCOS_FORMAT_ATTR_(printf, 2, 3);
                                                                                         ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_cmd.h:80:108: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_printf( VCOS_CMD_PARAM_T *param, const char *fmt, ... ) VCOS_FORMAT_ATTR_(printf, 2, 3);
                                                                                                            ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_cmd.h:80:90: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_printf( VCOS_CMD_PARAM_T *param, const char *fmt, ... ) VCOS_FORMAT_ATTR_(printf, 2, 3);
                                                                                          ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
./common/forbidden.h:57:89: error: expected ')' before 'SYMBOL'
 #define FORBIDDEN_SYMBOL_REPLACEMENT FORBIDDEN_look_at_common_forbidden_h_for_more_info SYMBOL !%*
                                                                                         ^
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:93: note: in definition of macro 'VCOS_FORMAT_ATTR_'
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                             ^
./common/forbidden.h:62:16: note: in expansion of macro 'FORBIDDEN_SYMBOL_REPLACEMENT'
 #define printf FORBIDDEN_SYMBOL_REPLACEMENT
                ^
/opt/vc/include/interface/vcos/vcos_cmd.h:81:118: note: in expansion of macro 'printf'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_vprintf( VCOS_CMD_PARAM_T *param, const char *fmt, va_list args ) VCOS_FORMAT_ATTR_(printf, 2, 0);
                                                                                                                      ^
In file included from /opt/vc/include/interface/vcos/vcos_types.h:38:0,
                 from /opt/vc/include/interface/vcos/vcos_assert.h:149,
                 from /opt/vc/include/interface/vcos/vcos.h:114,
                 from /opt/vc/include/interface/vmcs_host/vc_dispmanx.h:33,
                 from /opt/vc/include/EGL/eglplatform.h:110,
                 from /opt/vc/include/EGL/egl.h:36,
                 from ./backends/graphics/gles-custom/gles-custom.h:34,
                 from backends/graphics/gles-custom/gles-custom.cpp:25:
/opt/vc/include/interface/vcos/pthreads/vcos_platform_types.h:45:134: warning: 'FORBIDDEN_look_at_common_forbidden_h_for_more_info' is an unrecognized format function type [-Wformat=]
 #define VCOS_FORMAT_ATTR_(ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)  __attribute__ ((format (ARCHETYPE, STRING_INDEX, FIRST_TO_CHECK)))
                                                                                                                                      ^
/opt/vc/include/interface/vcos/vcos_cmd.h:81:100: note: in expansion of macro 'VCOS_FORMAT_ATTR_'
 VCOSPRE_ void VCOSPOST_ vcos_cmd_vprintf( VCOS_CMD_PARAM_T *param, const char *fmt, va_list args ) VCOS_FORMAT_ATTR_(printf, 2, 0);
                                                                                                    ^
distcc[12945] ERROR: compile backends/graphics/gles-custom/gles-custom.cpp on localhost failed
Makefile.common:103: recipe for target 'backends/graphics/gles-custom/gles-custom.o' failed
make: *** [backends/graphics/gles-custom/gles-custom.o] Error 1

´´´